servicecontrol/deploying-servicecontrol-in-a-cluster.md

```diff
@@ -29,7 +29,8 @@ More information on [Message Queuing in Server Clusters](https://technet.microso
 
 ## Database high availability
 
-The RavenDB database needs to be located in shared storage, highly available and fault tolerant. See [Customize RavenDB Embedded Location](configure-ravendb-location.md) for more information on how to change the ServiceControl database location.
+The RavenDB database needs to be located in *shared storage* that highly available and fault tolerant. Shared storage does not mean a shared network share but shared cluster storage. Access to the data should always be 'local' althought physically that data could be stored on a SAN.
+When this disk is mounted you need to configure RavenDB to use that location. See [Customize RavenDB Embedded Location](configure-ravendb-location.md) for more information on how to change the ServiceControl database location.
```

Comment:
Can you fix the following build error?

>Error: 'Invalid word(s) detected. (Invalid word(s) may be in a snippet(s).) The following strings are not allowed: you, we, our, your, us, please, yourself.
Invalid word detected on line 24: ' you '
    line: When this disk is mounted you need to configure RavenDB to use that location. See [Customize RavenDB Embedded Location](configure-ravendb-location.md) for more information on how to change the ServiceControl database location.